error: static assertion failed: Require 64bit time_t

I'm trying to compile the prod branch of Stellar-core on Linux Raspbian Jessie. GCC version 4.9 and clang 3.5.  I get the following error. Full logs attached: [stellarcorecompilelog.txt](https://github.com/stellar/stellar-core/files/1153204/stellarcorecompilelog.txt) 

In file included from ./database/Database.h:11:0,
                 from ./bucket/BucketApplicator.h:8,
                 from bucket/Bucket.cpp:10:
./util/Timer.h:63:5: error: static assertion failed: Require 64bit time_t
     static_assert(sizeof(uint64_t) == sizeof(std::time_t),
     ^

Is there a way to work around this? This is a 32 bit linux system, so there is no 64 bit time_t.... But I did not see 64 bit as a requirement for building.

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -25,6 +25,7 @@ When running a node, the best bet is to go with the latest release.
 - `pkg-config`
 - `bison` and `flex`
 - `libpq-devel` unless you `./configure --disable-postgres` in the build step below.
+- 64-bit system
 
 
 ### Ubuntu 14.04
@@ -44,9 +45,9 @@ When building on OSX, here's some dependencies you'll need:
 - brew install libtool
 - brew install automake
 - brew install pkg-config
-- brew install libpqxx *(If ./configure later complains about libpq missing, try PKG_CONFIG_PATH='/usr/local/lib/pkgconfig')* 
+- brew install libpqxx *(If ./configure later complains about libpq missing, try PKG_CONFIG_PATH='/usr/local/lib/pkgconfig')*
 
-### Windows 
+### Windows
 See [INSTALL-Windows.txt](INSTALL-Windows.txt)
 
 ## Basic Installation

